server/utils/jobPostingDateHandlers.js

```diff
@@ -0,0 +1,52 @@
+const moment = require('moment')
+const momentTz = require('moment-timezone')
+
+const { JobPosting } = require('../db/models')
+
+function validateDate(date) {
+  if (date === undefined || date === null) {
+    return null
+  }
+  const timeZone = 'Europe/Helsinki'
+  return momentTz.tz(date, 'YYYY-MM-DD', timeZone)
+}
+
+function formatDate(date) {
+  if (date === null) {
+    return null
+  }
+  let momentObj = moment(date, 'YYYY-MM-DD')
+  return momentObj.format('YYYY-MM-DD')
+}
+
+async function handleJobPostingsForAdmin() {
+  const jobpostings = await JobPosting.findAll({}).then(postings => postings.map(jobposting => jobposting.get({ plain: true })))
+
+  const now = moment().startOf('day')
+  jobpostings.forEach(jobposting => {
+    if (moment(now).isSameOrAfter(jobposting.showFrom) && moment(now).isSameOrBefore(jobposting.showTo)
```

Comment:
Joi has similar features for checking the date ranges: https://github.com/hapijs/joi/blob/v14.0.6/API.md#date---inherits-from-any